### DISCUSSION TITLE
All the features that Jupyter lab has for output cells

### DISCUSSION THREAD

**Discussion by stromal** [microsoft/vscode-jupyter#11353] [New Ideas]
I especially miss "enable scrolling for outputs".
But I would like to request that the team implement every features from jupyter lab.

![2022-09-12 112141](https://user-images.githubusercontent.com/19979901/189630923-f19c5b0f-1ed1-43a7-9a15-4824101cbc12.jpg)

**Comment by amunger** [comment]
scrolling for outputs is a fairly common request, I think https://github.com/microsoft/vscode-jupyter/issues/4406 is the best match for an issue.
